dali/base/dasds.cpp

```diff
@@ -2198,6 +2217,9 @@ void CDeltaWriter::addToQueue(CTransactionItem *item)
     }
     else // here if exceeded transationQueueLimit, transactionMaxMem or exceeded time threshold (deltaSaveThresholdSecs)
     {
+        if (isAborted()) // critical situation if delta writer is no longer running (there will have been previous errors)
```

Comment:
Why isAborted() rather than just testing aborted (here, but not elsewhere)?

Reply:
no particular reason, I'll change it so consistent.